src/main/java/com/bravo/user/dao/repository/PaymentRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.bravo.user.dao.repository;
+
+import com.bravo.user.dao.model.Payment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface PaymentRepository extends JpaRepository<Payment, String> {
```

Comment:
This doesn't appear to be tested in the test cases.

Reply:
Copy that. I will add test cases for the repository, as I always do, but forgot in this coding assessment. I appreciate the feedback - it really helps me grow fast.